app/user/rest.py

```diff
@@ -17,8 +17,13 @@
 @user.route('', methods=['POST'])
 def create_user():
     user, errors = user_schema.load(request.get_json())
+    req_json = request.get_json()
+    if not req_json.get('password'):
+        return jsonify(result="error", message={'error': 'password missing'}), 400
```

Comment:
Just to make this consistent with the marshmallow errors object, the dictionary value ('password missing') can that be a list.